File: deploy_model_as_api/sample.js
var formdata = new FormData();
formdata.append("file", fileInput.files[0], "<path to image file>");

var requestOptions = {
  method: 'POST',
  body: formdata,
  redirect: 'follow'
};

fetch("https://inference.navan.ai/predict?token=<your token>", requestOptions)
  .then(response => response.text())
  .then(result => console.log(result))
  .catch(error => console.log('error', error));

File: deploy_model_as_api/sample.py
import requests

url = "https://inference.navan.ai/predict?token=<your token>"

payload={}
files=[
  ('file',('<image file name>',open('<path to image file>','rb'),'image/jpeg'))
]
headers = {}

response = requests.request("POST", url, headers=headers, data=payload, files=files)

print(response.text)